react/test/containers/ParkFormContainer.js

```diff
@@ -28,8 +28,4 @@ describe('ParkFormContainer', () => {
   it('should render one input element', () => {
     expect(wrapper.find('input').length).toEqual(1);
   });
-
-  // it('should render a single input item', () => {
-  //   expect(wrapper.find('input').length).toEqual(1);
-  // });
 })
```

Comment:
Either get these tests running or remove this test (since you dont need to test components with Fetch in them)